wrappers/src/shared_realm_cs.cpp

```diff
@@ -754,6 +754,22 @@ REALM_EXPORT bool shared_realm_remove_type(const SharedRealm& realm, uint16_t* t
     });
 }
 
+REALM_EXPORT bool shared_realm_remove_all(const SharedRealm& realm, NativeException::Marshallable& ex)
+{
+    return handle_errors(ex, [&]() {
+        realm->verify_in_write();
+
+        realm::Group& all_tables = realm->read_group();
+        for(realm::TableKey table_key : all_tables.get_table_keys()) {
+            auto table = all_tables.get_table(table_key);
+            if (table->get_name().begins_with("class_")) {
```

Comment:
Thanks to conversation with Jørgen, this way only the classes get cleared and passes both tests 🎉